tests/FakeItEasy.Specs/ThrowsAsyncSpecs.cs

```diff
@@ -0,0 +1,310 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System;
+    using System.Threading.Tasks;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using Xbehave;
+
+    public class ThrowsAsyncSpecs
+    {
+        public interface IFoo
+        {
+            Task BarAsync();
+
+            Task BarAsync(int a);
+
+            Task BarAsync(int a, string b);
+
+            Task BarAsync(int a, string b, bool c);
+
+            Task BarAsync(int a, string b, bool c, double d);
+
+            Task<int> BazAsync();
+
+            Task<int> BazAsync(int a);
+
+            Task<int> BazAsync(int a, string b);
+
+            Task<int> BazAsync(int a, string b, bool c);
+
+            Task<int> BazAsync(int a, string b, bool c, double d);
+        }
+
+        [Scenario]
+        public void NoResultPassExceptionInstance(IFoo fake, Task task)
+        {
+            "Given a fake"
+                .x(() => fake = A.Fake<IFoo>());
+
+            "And an async method of the fake configured to throw asynchronously"
+                .x(() => A.CallTo(() => fake.BarAsync()).ThrowsAsync(new MyException()));
+
+            "When that method is called"

```

Comment:
Oh, man. Maybe not! Perhaps we've been at odds. I thought I remembered that from one of @adamralph's examples or comments, but I don't have that at hand. I think the text you have reads fine.